GetStream/stream-chat-react-native
# build: Run prettier for documentation with `lint` script

## 🎯 Goal

Currently, we run prettier on docs with `lint-fix` but nothing with `lint`. This adds `prettier --check` on `lint` so errors are caught in PRs.

## To test this

1. Make a change in a .mdx file within `docusaurus/`, that should be caught by the linter
2. run `yarn lint` in the root directory and see that it's caught
3. run `yarn lint-fix` and see that it fixes that error

## ☑️ Checklist

- [x] Commits follow the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) spec
